gh_pages/lifecycle.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: minimal
+---
+
+## Lifecycle
+
+You can think of respostatus as a way to describe the lifecycle-phase of a project. This may help you to choose an appropriate status for your repositories.
+
+Repostatus aimes to answer two questions about a repository:
+
+* How complete is this? Should I expect it to be "stable" software, mostly working, or not really useful at all?
+* Should I expect active development and support, maybe some bug fixes, or nothing at all?
+
+We wish to talk about whether the content of a repository is in a usable state and how much support from contributors can be expected. Thus statuses can be divided along two metrics: usableness and support.
+
+![Flowdiagram representation of a typical lifecycle.](/images/lifecycle.png)
+
+As illustrated above the statuses of a repository can be split into two groups: unstable repos and stable repos.
+
+### Unstable repos
+
+These repositories have not yet reached a stable and usable state. You should expect them to change frequently and dramatically.
+
+All repositories start as a Concept. They are little more than an idea yet. From here we might abandone the project or start to work on it in ernest. Only once a repository was a WIP can it be suspended, work on it be put aside t be continued later. To reach a stable state some work is needed thus WIP is the only state from which we can feasibly move towards a stable state.
```

Comment:
"abandone" -> "abandon"

Also, I think I'd change this to "All projects start as a Concept", just to account for the people who (scary, but some people do) start a project without version control.

Also looks like a typo, "put aside t be", "o" missing.

Reply:
✔️